exercises/tuples-lists-dictionaries/challenge_070.py

```diff
@@ -0,0 +1,44 @@
+# TUPLES, LISTS AND DICTIONARIES
+
+# Challenge 070
+"""
+70: Add to program 069 to ask the user a number and display the country in that position.
+"""
+
+country_names = ("England", "Italy", "Spain", "France", "Portugal", "Germany")
+
+print(", ".join(country_names))
+
+
+# Ask for user input and ensures the user enters an integer
+def get_user_input_digit(message):
+    while True:
+        user_input = input(message)
+        if not user_input.isdigit():
+            print("Oops! That was not a valid number. Try again...")
+        return int(user_input)
+
+
+def ask_for_country_by_index():
+    max_index = len(country_names) - 1
```

Comment:
What happens here if there are more than 10 country names though, does this code still work?

Reply:
Added more countries and restored the else statement